ocs_ci/ocs/platform_nodes.py

```diff
@@ -1685,6 +1686,67 @@ def read_default_config(self, default_config_path):
         return default_config_dict
 
 
+class IBMPowerNodes(NodesBase):
```

Comment:
Power has two platforms - libvirt and powervs.   This class should be updated to import modules to implement callouts for these two platforms.   libvirt sounds like it mght be standardized, but it is not really.   Each implementation may choose different names.   Power follows this pattern test-ocp-4-6-worker-0, but another implementation may utilize a different naming scheme.   On power, we may implement helper functions that invoke virsh or other functions.   Anyway, none of this would apply to other libvirt platforms as only power implements libvirt ocs-ci.   The powervs module should return not implemented for each callout.  Later, we can plug these with the proper callouts.

Reply:
Sure. For now, I am focussing on the libvirt environment. We can take up the PowerVS environment in a separate PR.

Reply:
The code still seems like it would try to invoke virsh on powervs.   What am I missing?